templates/journal/single_issue.html

```diff
@@ -4,104 +4,113 @@
+                {%- if issue.pdf %}
+                  {#-  -#}
+                  <a href="{{ url('download_issue', issue.number, 'pdf') }}"
+                  ><div class="download edge pdf nodrag"><span class="button-text">PDF</span></div></a>
+                  {#-  -#}
+                {% else -%}
+                  <div class="download edge greyedout nodrag"><span class="button-text">PDF</span></div>
+                  {#-  -#}
+                {% endif -%}
+                {% if issue.mobi %}
+                  {#-  -#}
+                  <a href="{{ url('download_issue', issue.number, 'mobi') }}"
+                  ><div class="download centre mobi nodrag"><span class="button-text">MOBI</span></div></a>
+                  {#-  -#}
+                {% else -%}
+                  <div class="download centre greyedout nodrag"><span class="button-text">MOBI</span></div>
+                  {#-  -#}
+                {% endif -%}
+                {%- if issue.epub %}{#-  -#}
+                  <a href="{{ url('download_issue', issue.number, 'epub') }}"
+                  ><div class="download edge epub nodrag"><span class="button-text">EPUB</span></div></a>
+                  {#-  -#}
+                {% else -%}
+                  <div class="download edge greyedout nodrag"><span class="button-text">EPUB</span></div>
+                  {#-  -#}
+                {% endif -%}
+                </div>
+            </div>
+          </div>
         </div>
-      </div>
     </div>
-  </div>
 
-  <div class="cover-panel" onmouseover="showCoverInfo()" onmouseout="hideCoverInfo()"  >
-    <p class="panel-heading">{{ issue.pretty_name }}</p>
+  <div class="cover-panel">
+    <p style="margin-bottom: 1.8rem" class="panel-heading">{{ issue.pretty_name }}</p>
 
-    {% if cover %}
-      <div id="cover-info">
-        <p id="cover-title">{{cover.title}}</p>
-        <p id="cover-artist">{{cover.artist}}</p>
-        <p><a target="_blank" href="mailto:{{ cover.artist_url }}">{{ cover.artist_url }}</a></p>
-      </div>
 
-      {% if cover.cover_image %}
-        <div id="issue-cover">{{ thumbnail(cover.cover_image, "320x464") }}</div>
+      {% if issue.pdf %}
+    <div id="issue-pdf"></div>
       {% endif %}
 
+
+      {% if cover %}
+        <div id="issue-cover">{{ thumbnail(cover.cover_image, "320x464") }}</div>
+
+
     {% else %}
       <div id="issue-cover" class="no-image"></div>
     {% endif %}
+
   </div>
 
 {% endblock %}
 
 {% block script %}
     <script>
 
-        function showCoverInfo(){
-            $("#cover-info").css("display","inline-block");
-
-        }
+if(PDFObject.supportsPDFs === false){
+   $("#issue-pdf").css("display","none");
+}
 
-        function hideCoverInfo(){
-            $("#cover-info").css("display", "none");
+        PDFObject.embed("{{issue.pdf.url}}", "#issue-pdf", {pdfOpenParams: { view: 'FitW', scrollbar: '1', toolbar: '0', statusbar: '1', messages: '0', navpanes: '1' }});
```

Comment:
At the moment this line breaks the site on this page if there is no PDF file uploaded for this issue. The script block needs to check whether there is a pdf file (`{% if issue.pdf %}{% endif %}`).